cktap/cli.py

```diff
@@ -628,12 +629,14 @@ def show_balance(cvc):
 
 @main.command('core')
 @click.option('--pretty', '-p', is_flag=True, help="Pretty-print JSON")
+@click.option('--sealed-only', '-so', is_flag=True, help="Only dump sealed slots")
```

Comment:
doubt "-so" works here, needs to be "--so" or something else, could be "-o"

Reply:
I was thinking about removing this option altogether as only one slot can be unsealed - so user should check which one is it and use --slot/-s N  - what do you think?

Reply:
sure, I like automatic